File: EndGate/EndGate.Core.Client.JS/Assets/Vectors/Helpers/MinMax.js
var EndGate;
(function (EndGate) {
    (function (Core) {
        (function (Assets) {
            var MinMax = (function () {
                function MinMax(min, max) {
                    this.Min = min;
                    this.Max = max;
                }
                return MinMax;
            })();
            Assets.MinMax = MinMax;            
        })(Core.Assets || (Core.Assets = {}));
        var Assets = Core.Assets;
    })(EndGate.Core || (EndGate.Core = {}));
    var Core = EndGate.Core;
})(EndGate || (EndGate = {}));

File: EndGate/EndGate.Core.Net/Assets/Vectors/Helpers/MinMax.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace EndGate.Core.Net.Assets
{
    public class MinMax
    {
        public MinMax(Number min, Number max)
        {
            Min = min;
            Max = max;
        }

        public Number Min { get; set; }
        public Number Max { get; set; }
    }
}
